<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Christoph Eckert</author>
 <class>IOConfig</class>
 <widget class="QDialog" name="IOConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="HelpLabel">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="2" column="1">
      <widget class="QComboBox" name="ComboBoxCharSet">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="LabelType">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LabelCharSet">
       <property name="text">
        <string>Characterset:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LabelValue">
       <property name="text">
        <string>Path/Device:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="ComboBoxOptions">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LabelOptions">
       <property name="text">
        <string>Options:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ComboBoxType">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="ComboBoxPath">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="BtnSelectFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to select a file from your filesystem</string>
       </property>
       <property name="statusTip">
        <string>Click to select a file from your filesystem</string>
       </property>
       <property name="whatsThis">
        <string>Click to select a file from your filesystem</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../GPSBook/icons/silk/silk_icons.qrc">
         <normaloff>:/icons/silk/folder_page_white.png</normaloff>:/icons/silk/folder_page_white.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="BtnSelectFile_2">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to select a file from your filesystem</string>
       </property>
       <property name="statusTip">
        <string>Click to select a file from your filesystem</string>
       </property>
       <property name="whatsThis">
        <string>Click to select a file from your filesystem</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../GPSBook/icons/silk/silk_icons.qrc">
         <normaloff>:/icons/silk/table_go.png</normaloff>:/icons/silk/table_go.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="1" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Use GPSBook current trace</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>360</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ComboBoxType</tabstop>
  <tabstop>ComboBoxOptions</tabstop>
  <tabstop>ComboBoxCharSet</tabstop>
  <tabstop>ComboBoxPath</tabstop>
  <tabstop>BtnSelectFile</tabstop>
  <tabstop>ButtonOk</tabstop>
  <tabstop>ButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../GPSBook/icons/silk/silk_icons.qrc"/>
 </resources>
 <connections/>
</ui>
